blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -6,44 +6,45 @@
    "metadata": {},
```

Comment:
<p>Should we use a model that is not sampling so awfully? Seems distracting for no reason</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Yeah... also one where the JAX speed-up is bigger would be good. Any ideas?

Reply:
Something along the lines of the benchmark model in the blogpost?

Reply:
Actually it's kinda nice because the different init at the end fixes the divergences. Maybe we can just remove a bit the clutter in the output.

Reply:
It still renders the time comparisons before pretty meaningless. Who cares that a model that is sampling bad is faster in one backend than the other?

Reply:
I increased target_accept to 0.9 which fixed it.